src/robot_manager.cpp

```diff
@@ -753,7 +784,14 @@ catch(resource_registrar::TemotoErrorStack& e)
 bool RobotManager::goalNavigationCb(RobotNavigationGoal::Request& req, RobotNavigationGoal::Response& res)
 try
 {
+  TEMOTO_INFO_("Received a goal Navigation request");
+  TEMOTO_DEBUG_STREAM_("Request:\n" << req);
+
```

Comment:
the priority management is completely missing, refer to the implementation of custom feature